### DISCUSSION TITLE
Missing detection of CRAN package

### DISCUSSION THREAD

**Discussion by rubak** [r-universe-org/help#218] [Q&A] [answered]
At the [spatstat universe](https://spatstat.r-universe.dev/ui#packages) one package `spatstat.model` has a question mark indicating that it may or may not coincide with the CRAN package. This is different than our other packages. The `DESCRIPTION` file has:

```
BugReports: https://github.com/spatstat/spatstat.model/issues
```

This is the same structure as our other packages and they appear as verified on CRAN. Any idea why this is different?

I guess one solution is to add `https://github.com/spatstat/spatstat.model/` under `URL`, but I think we may prefer to just keep this field as a single link to our website https://spatstat.org, and as I mentioned it seems to work without this for the other packages (e.g. `spatstat.explore`).

PS: Somewhere I should say a big THANK YOU for the r-universe service. It is fantastic!

**Comment by jeroen** [comment, marked as answer]
Actually the `DESCRIPTION` file of the CRAN version of your package still refers to `spatstat.core` instead of `spatstat.model` in the BugReports field. You seem to have [fixed this recently](https://github.com/spatstat/spatstat.model/commit/6ef31f652e177f032db0b2618cc08b7b43b3e6d1) but you have not published that to CRAN yet, as you can see from the CRAN homepage: https://cran.r-project.org/web/packages/spatstat.model/index.html

Therefore the system still thinks of https://github.com/spatstat/spatstat.core as the official url. The problem will resolve itself once you push a new version of  spatstat.model with https://github.com/spatstat/spatstat.model/commit/6ef31f652e177f032db0b2618cc08b7b43b3e6d1 to cran...

**Reply by rubak** [reply]
You are of course right. Forgot to think about this not being on CRAN. Sorry about that.
